### PR TITLE
Update test_System.py

### PR DESCRIPTION
**Pull request by Mnlohani**
Added some test cases.

### CODE REVIEW

File: Particle_Simulation/test/test_System.py [hunk 1]
```diff
@@ -1,42 +1,213 @@
...
+        #particle_position_out_of_box
+        particle_positions = np.array([[0.0, 0.5], [1.5, 4.5], [2.5, 2.5], 
+                                       [3.0, 4.5], [4.5, 5.5], [5.5, 3.5],
+                                       [6.0, 5.0], [6.5, 9.5], [7.0, 1.1], 
+                                       [8.25, 2.0], [9.75,12.0], [10.0 ,8.0 ] ])
+        box_space = np.array([10.5 , 11.0 ])
+        cutoff = 3.5
+        s1 = System(particle_positions, box_space, cutoff)
+        s1.construct_neighborlist() 
+
```
Comment (BioFreak95) [inline Particle_Simulation/test/test_System.py]: What did you want to test here?

File: Particle_Simulation/test/test_System.py [hunk 2]
```diff
@@ -1,42 +1,213 @@
...
+        particle_positions = np.array([[0.0, 0.5], [1.5, 4.5], [2.5, 2.5], 
+                                       [3.0, 4.5], [4.5, 5.5], [5.5, 3.5],
+                                       [6.0, 5.0], [6.5, 9.5], [7.0, 1.1], 
+                                       [8.25, 2.0], [9.75,11.0], [10.0 ,8.0 ] ])
+        box_space = np.array([10.5 , 11.0, 2.0])
+        self.assertEqual(len(particle_positions), len(box_space), 'Fail: dim not equal')
+        cutoff = 3.5
+        s1 = System(particle_positions, box_space, cutoff)
+        s1.construct_neighborlist()
+
```
Comment (BioFreak95) [inline Particle_Simulation/test/test_System.py]: What did you want to test here?

File: Particle_Simulation/test/test_System.py [hunk 3]
```diff
@@ -1,42 +1,213 @@
...
+                                       [6.0, 5.0, 11], [6.5, 9.5, 10.5], [7.0, 1.1, 0.2], 
+                                       [8.25, 2.0, 1.0], [9.75, 9.0, 10.5], [10.0 ,8.0, 10.5 ] ])
+        box_space = np.array([10.5 , 11.0, 10.5 ])
+        cutoff = 3.5
+        refrence_cell_no = 27 
+        s1 = System(particle_positions, box_space, cutoff)
+        s1.construct_neighborlist()
+        cell_list_length =len(s1.cell_list)
+        self.assertEqual(refrence_cell_no, cell_list_length , 'Fail: CellList length not equals reference') 
+
```
Comment (BioFreak95) [inline Particle_Simulation/test/test_System.py]: What did you want to test here?

**Comment by Mnlohani** [comment]
ok.. I have added few new. closing this pull request.

**Pull request closed without merging**